pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/kubelet/component_test.go

```diff
@@ -265,8 +265,18 @@ function kubelet_monitoring {
       continue
     fi
 
-    node_object="$(kubectl get nodes -l kubernetes.io/hostname=$(hostname) -o json)"
-    node_status="$(echo $node_object | jq -r '.items[0].status')"
+    node_name=
+    if [[ ! -s "/var/lib/kubelet/nodename" ]]; then
+      node_name="$(cat "/var/lib/kubelet/nodename")"
```

Comment:
I'm not sure if the negation is intended in this condition.

`-s file` from "man bash"
> True if file exists and has a size greater than zero.